packages/suite-native-atoms/src/Input/Input.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactNode, useRef, useState } from 'react';
+import { TouchableWithoutFeedback, TextInput } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Text } from '../Text';
+import { Box } from '../Box';
+
+type InputProps = {
+    value: string;
+    label: string;
+    onChange: (value: string) => void;
+    isDisabled?: boolean;
+    isInvalid?: boolean;
+    hasWarning?: boolean;
+    leftIcon?: ReactNode;
+};
+
+type InputWrapperStyleProps = {
+    hasWarning: boolean;
+    isInvalid: boolean;
+};
+const inputWrapperStyle = prepareNativeStyle<InputWrapperStyleProps>(
+    (utils, { isInvalid, hasWarning }) => ({
+        borderWidth: utils.borders.widths.small,
+        borderColor: utils.colors.gray300,
+        backgroundColor: utils.colors.gray300,
+        borderRadius: utils.borders.radii.small,
+        paddingVertical: utils.spacings.small,
+        paddingHorizontal: 14,
+        height: 58,
+        justifyContent: 'center',
+        extend: [
+            {
+                condition: hasWarning,
+                style: {
+                    borderColor: utils.colors.yellow,
+                    borderWidth: utils.borders.widths.large,
+                },
+            },
+            {
+                condition: isInvalid,
+                style: {
+                    borderColor: utils.colors.red,
```

Comment:
I think there should be also some bg color according to Figma

Reply:
[7514c50](https://github.com/trezor/trezor-suite/pull/5583/commits/7514c50b146c1b5e466cfff4f4fd818477d27457)